src/playlist-loader.js

```diff
@@ -14,7 +14,9 @@ import {
   addPropertiesToMain,
   mainForMedia,
   setupMediaPlaylist,
-  forEachMediaGroup
+  forEachMediaGroup,
```

Comment:
Is this used anywhere?

Reply:
Yeah, just [not in my changes](https://github.com/videojs/http-streaming/blob/5534ec726ab626fafb8ea2b95882a9a9b20a4c17/src/playlist-loader.js#L335) 👍

Reply:
Oh nice, good catch!